packages/mobile/src/identity/verification.ts

```diff
@@ -417,11 +419,19 @@ function* revealAndCompleteAttestation(
   account: string,
   e164Number: string,
   e164NumberHash: string,
-  issuer: string
+  issuer: string,
+  index: number
 ) {
   Logger.debug(TAG + '@revealAttestation', `Revealing an attestation for issuer: ${issuer}`)
   CeloAnalytics.track(CustomEventNames.verification_reveal_attestation, { issuer })
   const revealTx = yield call(makeRevealTx, attestationsContract, e164Number, issuer)
+  // Crude way to prevent sendTransaction being called in parallel and use the same nonces.
+  const waitBeforeSendTransactionInMs: number = index * 100
```

Comment:
Not sure about this approach but given that we'll soon be removing the reveal tx, I guess it's fine.
A more robust approach would be to build the synchronization and/or delay into the `sendTransaction` utility function

Reply:
Removed. This isn't required anymore.